options/options.go

```diff
@@ -31,6 +32,7 @@ type ProvenanceOpts struct {
 
 // BuildOpts are the options for checking the builder.
 type BuilderOpts struct {
-	// ExpectedID is the expected builder ID.
+	// ExpectedID is the expected builder ID that is inputted
+	// at the command line by the user to be verified.
```

Comment:
nit: inputted just sounds weird
```suggestion
	// ExpectedID is the expected builder ID that is provided by the user.
```

Reply:
changed =)